### PR TITLE
add leticiarosario

### LINKED ISSUE #554: Leticia Rosario
leticiarosario

### PR DESCRIPTION
**Pull request by Leticiarosario**
resolve #554

### CODE REVIEW

**Comment by starlightknown** [comment]
Hey just for information, you don't need to open a new PR everytime when you want to change something, it can be done in the same PR

File: data.json [hunk 1]
```diff
@@ -995,4 +995,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/10162710/120894576-6ff8f200-c619-11eb-9ee5-5078912d5550.png",
     "issueNumber": 552
   }
```
Comment (bishalr0y) [inline data.json:997]:
```suggestion
  },
```

**Review by bishalr0y — CHANGES_REQUESTED** [review]

**Comment by schmelto** [comment]
> Hey just for information, you don't need to open a new PR everytime when you want to change something, it can be done in the same PR, also you missed to do the changes in this PR. It's same as the last one

To do so add a commit in the related branch with the requested changes or simply click on the button "Commit suggestion" this will get the suggested changes and make a commit to your branch and PR.

image

You can chose which PR you want to continue. Open are #555, #556 and #557.

Kindly commit the relevant changes in one of those and close the other two.

If you need any help please let us know.

**Review by schmelto — CHANGES_REQUESTED** [review]
please resolve the issue @bishalr0y has reviewed

**Pull request closed without merging**